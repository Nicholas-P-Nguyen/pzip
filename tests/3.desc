test error handling (no files)
