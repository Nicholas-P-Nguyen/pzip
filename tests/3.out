pzip: file num_threads
